src/main/java/com/parsing/rest/ParserResource.java

```diff
@@ -0,0 +1,23 @@
+package com.parsing.rest;
+
+import com.parsing.parsers.parserlot.ParserLotService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import java.io.IOException;
+
+
+@RestController
+@RequiredArgsConstructor
+public class ParserResource {
+
+    private final ParserLotService parserLotService;
+
+    @GetMapping("/parser_prozorro")
+    @ResponseStatus(HttpStatus.FOUND)
+    public void parsing () throws IOException {
```

Comment:
shouldn't be any throws here, use try-catch and re-throw some RuntimeException please

Reply:
I added RuntimeException but this case needs IOException too.